src/gui/MainWindow.cpp

```diff
@@ -168,6 +173,13 @@ MainWindow::MainWindow()
     , m_appExiting(false)
 {
     m_ui->setupUi(this);
+    
+#if defined(WITH_XC_DBUS) && !defined(QT_NO_DBUS)
+    new MainWindowAdaptor(this);
+    QDBusConnection dbus = QDBusConnection::sessionBus();
+    dbus.registerObject("/keepassxc", this);
+    dbus.registerService("org.keepassxc.MainWindow");
```

Comment:
This should be `org.keepassxc.KeePassXC.MainWindow`.

Reply:
Seems redundant using the name 2 times

Reply:
That's how the standard works.